arch/arm/boot/dts/imx6qdl-ts4900-ts8950.dtsi

```diff
@@ -0,0 +1,339 @@
+		pendown-gpio = <&gpio4 25 GPIO_ACTIVE_HIGH>;
+		vcc-supply = <&reg_3v3>;
+		ti,swap-xy;
+		ti,keep-vref-on;
+		ti,vref-mv = <3300>;
+		ti,settle-delay-usec = /bits/ 16 <500>;
+		ti,x-plate-ohms = /bits/ 16 <715>;
+		ti,pressure-max = /bits/ 16 <65535>;
+		ti,debounce-rep = /bits/ 16 <1>;
+		ti,debounce-tol = /bits/ 16 <30>;
+		ti,debounce-max = /bits/ 16 <255>;
+		ti,pendown-gpio-debounce = <5000>;
+		wakeup-source;
+	};
+
+	spidev: spi@2 {
+		compatible = "spidev";
+		reg = <2>;
+		spi-max-frequency = <18000000>;
+	};
+};
+
+/*
+ * This platform uses a different pin for USB 5 V control than other
+ * combinations and what the TS-4900 dts expects. Because of that,
+ * need to adjust pin names for sanity.
+ */
+
+&gpio2 {
+	gpio-line-names = "", "", "", "", "", "", "", "", "", "", "", "", "",
+		"", "", "DIO_2", "DIO_18", "DIO_12", "DIO_16", "EN_LCD_3.3V",
+		"DIO_14", "OFF_BD_RESET#", "", "BUS_CS#", "", "DIO_15", "BUS_DIR",
+		"BUS_ALE", "EN_SD_POWER#", "DIO_10", "", "";
+};
+
+&gpio4 {
+	gpio-line-names = "", "", "", "", "", "DIO_7", "", "", "", "", "", "",
+		"", "", "", "", "HOME_P_SW#", "LCD_DE", "LCD_HSYNC", "LCD_VSYNC",
+		"", "LCD_D00", "EN_USB_5V#", "LCD_D02", "EN_GPS_3V3#", "TOUCH_WAKE#",
+		"LCD_D05", "LCD_D06", "POWER_FAIL#", "LCD_D08", "LCD_D08",
+		"LCD_D10";
+};
+
+&gpio5 {
+	gpio-line-names = "EIM_WAIT#", "", "", "", "DIO_20", "LCD_D11",
+		"LCD_D12", "LCD_D13", "LCD_D14", "LCD_D15", "BACK_P_SW#",
+		"LCD_D17","LCD_D18", "USER_JP#", "LCD_D20", "LCD_D21",
+		"LCD_D22", "LCD_D23", "DIO_9", "DIO_8", "FPGA_DONE",
+		"FPGA_RESET#", "AUD_CLK", "AUD_TXD", "AUD_FRM", "AUD_RXD",
+		"", "", "", "", "DIO_0", "DIO_6";
+
+	en-line-amp-hog {
+		gpio-hog;
+		gpios = <15 GPIO_ACTIVE_HIGH>;
+		line-name = "LINE_AMP_EN";
+		output-high;
+	};
+};
+
+&i2c2 {
+	status = "okay";
+	clock-frequency = <100000>;
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_i2c2>;
+
+	sgtl5000: audio-codec@a {
+		compatible = "fsl,sgtl5000";
+		pinctrl-names = "default";
+		pinctrl-0 = <&pinctrl_sgtl5000>;
+		reg = <0x0a>;
+		clocks = <&clks IMX6QDL_CLK_CKO>;
+		VDDA-supply = <&reg_3v3>;
+		VDDIO-supply = <&reg_3v3>;
+	};
+
+	mma8451: accelerometer@1c {
+		compatible = "fsl,mma8451";
+		reg = <0x1c>;
+		vdd-supply = <&reg_3v3>;
+		vddio-supply = <&reg_3v3>;
+		interrupt-parent = <&gpio5>;
+		interrupts = <17 IRQ_TYPE_LEVEL_LOW>;
+		interrupt-names = "INT1";
+		/* INT2 connected to spare pins, otherwise unused */
+	};
+};
+
+&ldb {
+	status = "okay";
+
+	lvds-channel@0 {
+		crtc = "ipu1-di1";
+		fsl,data-mapping = "spwg";
+		fsl,data-width = <18>;
+		status = "okay";
+		primary;
+
+	};
+};
+
+&iomuxc {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_hog &pinctrl_lcd_gpio>;
+
+	pinctrl_ecspi2_touch: ecspi2 {
+		fsl,pins = <
+			MX6QDL_PAD_CSI0_DAT8__ECSPI2_SCLK      0x100b1
+			MX6QDL_PAD_CSI0_DAT9__ECSPI2_MOSI      0x100b1
+			MX6QDL_PAD_CSI0_DAT10__ECSPI2_MISO     0x100b1
+			MX6QDL_PAD_CSI0_DAT11__GPIO5_IO29      0x100b1   // Offboard CS0#
```

Comment:
minor nitpick, but the whitespace is different for just 2 comments here

Reply:
Cleaned up all spacing